src/partials/external_authentication_required.hbs

```diff
@@ -0,0 +1,18 @@
+<div class="card">
+  <div class="org-logo">
+    <img class="org-logo__image" src="{{logo}}" alt="Company Logo"/>
+  </div>
+  <h1 class="heading">Authenticate In Popup</h1>
+  <div id="spinnerId" style="display:block">
```

Comment:
Can we use the spinner handlbar here? {{> spinner}}

Reply:
I want it to be displayed when it is rendered, the spinner handlebar is not displayed by default.

Reply:
Gotcha, yeah I see now that the spinner code works in the opposite way. This seems fine